source/elements/oneTBB/source/containers/concurrent_bounded_queue_cls.rst

```diff
@@ -113,6 +124,37 @@ Member functions
     concurrent_bounded_queue_cls/unsafe_member_functions.rst
     concurrent_bounded_queue_cls/iterators.rst
 
+Non-member functions
+--------------------
+
+This function provides swap operation on ``oneapi::tbb::concurrent_queue``
```

Comment:
It seems like it is the wrong place for this sentence - it describes only `swap` and this section also covers comparisons

Reply:
fixed